keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -331,7 +331,7 @@ metadata:
   serviceAccountName: [5]
   accessSecretName: [6]
   provisioner: [7]
-  policyActions [8]
+  policyActionsConfigMapData: [8]
```

Comment:
Yes, this names is good. A possibly shorter name that is still meaningful could be `policyActionsData`, but this name loses the association to a CM.